src/lib/parse-ical-data.ts

```diff
@@ -18,25 +71,41 @@ export interface AcmEvent {
   isHappening: boolean;
 }
 
+const convertIcalDateTime = (datetime: string): Date => {
+  const characters = datetime.split("");
+  const fullYear = characters.splice(0, 4).join("");

```

Comment:
`characters.splice(a, b).join("")` could be `datetime.slice(a, b)` instead without splitting `datetime`.

Reply:
That would possibly be more readable. It would indeed work since the numbers in the iCal datetime format are always in the same spots.

Reply:
Fixed: <https://github.com/EthanThatOneKid/acmcsuf.com/pull/105/files#diff-c9b7c8232c884e8d6c62a724e8cf309ba0d316ca3bf4cad5a1515820a0911edeR76-R81>